File: src/codes/Users_Manual/device_1.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~using namespace autd3;
//~auto autd =
//~Controller::open({AUTD3{}}, link::Nop{});
//~{
auto tr = autd[0][0];
//~}
//~{
for (auto& tr : autd[0]) {
  // do something
}
//~}
//~return 0; }

File: src/codes/Users_Manual/device_1.rs
# use autd3::prelude::*;
# #[allow(unused_variables)] 
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let autd = Controller::open([AUTD3::default()], autd3::link::Nop::new())?;
let tr = &autd[0][0];
for tr in &autd[0] {
    // do something
}
# Ok(())
# }